java/src/org/openqa/selenium/grid/commands/Standalone.java

```diff
@@ -213,33 +219,7 @@ protected Handlers createHandlers(Config config) {
     // Allow the liveness endpoint to be reached, since k8s doesn't make it easy to authenticate
     // these checks
     httpHandler = combine(httpHandler, Route.get("/readyz").to(() -> readinessCheck));
-
-    Node node = new NodeOptions(config).getNode();
-    combinedHandler.addHandler(node);
-    distributor.add(node);
-
-    bus.addListener(
-        NodeDrainComplete.listener(
-            nodeId -> {
-              if (!node.getId().equals(nodeId)) {
-                return;
-              }
-
-              // Wait a beat before shutting down so the final response from the
-              // node can escape.
-              new Thread(
-                      () -> {
-                        try {
-                          Thread.sleep(1000);
-                        } catch (InterruptedException e) {
-                          // Swallow, the next thing we're doing is shutting down
-                        }
-                        LOG.info("Shutting down");
-                        System.exit(0);
-                      },
-                      "Standalone shutdown: " + nodeId)
-                  .start();
-            }));
+    Node node = createNode(config, bus, distributor, combinedHandler);
```

Comment:
I understand why this is done. But it would be nice if such changes were a different PR. It makes it easier and shorter to review this one :) Just a suggestion.